rero_ils/modules/selfcheck/api.py

```diff
@@ -273,10 +286,11 @@ def item_information(item_barcode, **kwargs):
             SelfcheckItemInformation, SelfcheckSecurityMarkerType
         org_pid = kwargs.get('institution_id')
         item = Item.get_item_by_barcode(item_barcode, org_pid)
-        with current_app.test_request_context() as ctx:
-            language = kwargs.get('language', current_app.config
-                                  .get('BABEL_DEFAULT_LANGUAGE'))
-            ctx.babel_locale = language
+        language = kwargs.get('language', current_app.config
+                              .get('BABEL_DEFAULT_LANGUAGE'))
+        # Temporarily overrides the currently selected locale.
+        # force_locale is allowed to work outside of application context
+        with force_locale(language):
```

Comment:
Maybe better to change to:
```python
if item:
    with force_locale(language):
        document = Document.get_record_by_pid(item.document_pid)
        location = item.get_location()
```

Reply:
I need `force_locale` for the else statement